suitesparse_bindings/suitesparse-src/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "suitesparse-src"
+version = "0.1.0"
```

Comment:
Just a thought: should this version match the suitesparse version?

Reply:
I find it's easier to leave these two versions separate, and rather have an easy way to find the linked version, pushed a new commit for this. This way it is easier to introduce breaking changes, which we will probably have to do as we iterate on this design

Reply:
Ok that's a good rationale.